docs/package.json

```diff
@@ -77,6 +77,7 @@
     "motion": "^10.18.0",
     "next": "^15.2.0",
     "next-themes": "^0.3.0",
+    "oniguruma-to-js": "^0.4.3",
```

Comment:
I added this dependency because better-auth docs required it even tho it was not in the package.json dependency list. I may remove it if this is already fixed